From bc8a4a04aec1c52ffd28b17ef593f61ef2d84e0d Mon Sep 17 00:00:00 2001
From: Bruce Leidl <bruce@subgraph.com>
Date: Thu, 22 Sep 2022 19:57:38 -0400
Subject: [PATCH] moz.configure do not look for llvm-dump

---
 moz.configure | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/moz.configure b/moz.configure
index 8b74afbabe..d43afbaf2b 100755
--- a/moz.configure
+++ b/moz.configure
@@ -785,15 +785,15 @@ def llvm_tool(name):
     return llvm_tool
 
 
-llvm_objdump = check_prog(
-    "LLVM_OBJDUMP",
-    llvm_tool("llvm-objdump"),
-    what="llvm-objdump",
-    when="--enable-compile-environment",
-    paths=clang_search_path,
-)
-
-add_old_configure_assignment("LLVM_OBJDUMP", llvm_objdump)
+#llvm_objdump = check_prog(
+#    "LLVM_OBJDUMP",
+#    llvm_tool("llvm-objdump"),
+#    what="llvm-objdump",
+#    when="--enable-compile-environment",
+#    paths=clang_search_path,
+#)
+#
+#add_old_configure_assignment("LLVM_OBJDUMP", llvm_objdump)
 
 
 @depends(llvm_tool("llvm-readelf"), toolchain_prefix)
