From ac70fbe1edeef149c55d5779008429aec9945b76 Mon Sep 17 00:00:00 2001
From: Bruce Leidl <bruce@subgraph.com>
Date: Tue, 5 Oct 2021 11:17:47 +0000
Subject: [PATCH] Disabled calendar events from user session

---
 js/ui/sessionMode.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/js/ui/sessionMode.js b/js/ui/sessionMode.js
index afcd67c..b686efb 100644
--- a/js/ui/sessionMode.js
+++ b/js/ui/sessionMode.js
@@ -71,7 +71,7 @@ const _modes = {
 
     'user': {
         hasOverview: true,
-        showCalendarEvents: true,
+        showCalendarEvents: false,
         showWelcomeDialog: true,
         allowSettings: true,
         allowExtensions: true,
