From 87a40c18adf3305f8b82679cc80b544158b566f9 Mon Sep 17 00:00:00 2001
From: Bruce Leidl <bruce@subgraph.com>
Date: Fri, 27 May 2022 14:51:48 -0400
Subject: [PATCH] Patch locking logout so it does not disable shutting down

---
 gnome-session/gsm-manager.c | 15 +++++++--------
 1 file changed, 7 insertions(+), 8 deletions(-)

diff --git a/gnome-session/gsm-manager.c b/gnome-session/gsm-manager.c
index 1b88b26..fb3f2ec 100644
--- a/gnome-session/gsm-manager.c
+++ b/gnome-session/gsm-manager.c
@@ -560,10 +560,10 @@ end_phase (GsmManager *manager)
         case GSM_MANAGER_PHASE_APPLICATION:
                 break;
         case GSM_MANAGER_PHASE_RUNNING:
-                if (_log_out_is_locked_down (manager)) {
+                /*if (_log_out_is_locked_down (manager)) {
                         g_warning ("Unable to logout: Logout has been locked down");
                         start_next_phase = FALSE;
-                }
+                }*/
                 break;
         case GSM_MANAGER_PHASE_QUERY_END_SESSION:
                 if (!do_query_end_session_exit (manager))
@@ -2633,13 +2633,13 @@ gsm_manager_shutdown (GsmExportedManager    *skeleton,
                 return TRUE;
         }
 
-        if (_log_out_is_locked_down (manager)) {
+        /*if (_log_out_is_locked_down (manager)) {
                 g_dbus_method_invocation_return_error (invocation,
                                                        GSM_MANAGER_ERROR,
                                                        GSM_MANAGER_ERROR_LOCKED_DOWN,
                                                        "Logout has been locked down");
                 return TRUE;
-        }
+        }*/
 
         task = g_task_new (manager, priv->end_session_cancellable, (GAsyncReadyCallback) complete_end_session_task, invocation);
 
@@ -2669,13 +2669,13 @@ gsm_manager_reboot (GsmExportedManager    *skeleton,
                 return TRUE;
         }
 
-        if (_log_out_is_locked_down (manager)) {
+        /*if (_log_out_is_locked_down (manager)) {
                 g_dbus_method_invocation_return_error (invocation,
                                                        GSM_MANAGER_ERROR,
                                                        GSM_MANAGER_ERROR_LOCKED_DOWN,
                                                        "Logout has been locked down");
                 return TRUE;
-        }
+        }*/
 
         task = g_task_new (manager, priv->end_session_cancellable, (GAsyncReadyCallback) complete_end_session_task, invocation);
 
@@ -2697,8 +2697,7 @@ gsm_manager_can_shutdown (GsmExportedManager    *skeleton,
 
         g_debug ("GsmManager: CanShutdown called");
 
-        shutdown_available = !_log_out_is_locked_down (manager) &&
-                (gsm_system_can_stop (priv->system)
+        shutdown_available = (gsm_system_can_stop (priv->system)
                  || gsm_system_can_restart (priv->system)
                  || gsm_system_can_suspend (priv->system)
                  || gsm_system_can_hibernate (priv->system));
