From eadb72985d59d66684225591ec395610a3877c0b Mon Sep 17 00:00:00 2001
From: Bruce Leidl <bruce@subgraph.com>
Date: Thu, 22 Sep 2022 18:01:51 -0400
Subject: [PATCH] Don't require ssh binaries for build

---
 meson.build | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/meson.build b/meson.build
index ad41582..d16ef48 100644
--- a/meson.build
+++ b/meson.build
@@ -58,8 +58,8 @@ if p11_system_config_modules == ''
   error('Couldn\'t find location for pkcs11 module config')
 endif
 libsecret_dep = dependency('libsecret-1', version: '>= 0.20', required: get_option('ssh_agent'))
-ssh_add_path = find_program('ssh-add', required: get_option('ssh_agent')).full_path()
-ssh_agent_path = find_program('ssh-agent', required: get_option('ssh_agent')).full_path()
+#ssh_add_path = find_program('ssh-add', required: get_option('ssh_agent')).full_path()
+#ssh_agent_path = find_program('ssh-agent', required: get_option('ssh_agent')).full_path()
 
 with_systemd = false
 libsystemd_deps = []
@@ -96,8 +96,8 @@ conf.set('HAVE_TIMEGM', cc.has_function('timegm'))
 conf.set('HAVE_MLOCK', cc.has_function('mlock'))
 conf.set_quoted('GPG_EXECUTABLE', gpg_path)
 conf.set_quoted('LIBGCRYPT_VERSION', libgcrypt_dep.version())
-conf.set_quoted('SSH_ADD_EXECUTABLE', ssh_add_path)
-conf.set_quoted('SSH_AGENT_EXECUTABLE', ssh_agent_path)
+#conf.set_quoted('SSH_ADD_EXECUTABLE', ssh_add_path)
+#conf.set_quoted('SSH_AGENT_EXECUTABLE', ssh_agent_path)
 conf.set10('WITH_SYSTEMD', with_systemd)
 config_file = configure_file(
   output: 'config.h',
